.github/workflows/ci.yml

```
@@ -203,6 +220,124 @@ jobs:
+      - name: Install dependencies
+        run: uv sync --all-extras
+
+      - name: Build distributions
+        run: uv build
+
+      - name: Dependency integrity check
+        run: uv run python -m pip check
+
+      - name: Import smoke test
+        run: uv run python - <<'PY'
+import flujo
```

actionlint (1.7.9)

could not parse as YAML: could not find expected ':'

(syntax-check)